src/tools/CMakeLists.txt

```diff
@@ -20,6 +22,8 @@ install(TARGETS ceph_scratchtoolpp DESTINATION bin)
 
 add_executable(ceph_radosacl radosacl.cc)
 target_link_libraries(ceph_radosacl librados global)
+set_target_properties(ceph_radosacl PROPERTIES
```

Comment:
@rjfd why you don't set the `INSTALL_RPATH` for all executables? and what makes `ceph_radosacl`, `ceph_scratchtool` and `ceph_test_libcephfs` different from others?